README.md

```diff
@@ -12,69 +12,13 @@ npm install --save-dev @enact/jsdoc-to-ts
 
 ## Usage
 
-Assuming this directory is a peer of the Enact source and you want to write this into an installed (from npm) version of enact:
+Assuming this directory is a peer of the Sandstone source and you want to write this into a same path of sandstone:
 
 ```bash
-node -e "['core', 'ui', 'moonstone', 'i18n', 'webos', 'spotlight'].forEach(p => require('.')({
-  package: '../enact/packages/' + p,
-  output: require('fs').writeFileSync,
-  ignore: ['node_modules', 'ilib'],
-  importMap: {
-    ui: '@enact/ui',
-    moonstone: '@enact/moonstone',
-    core: '@enact/core',
-    webos: '@enact/webos',
-    spotlight: '@enact/spotlight',
-    i18n: '@enact/i18n'
-  },
-  outputPath: '<path to installation>/node_modules/@enact/' + p
-}))"
+node bin/jsdoc-to-ts.js ../sandstone -o=../sandstone
```

Comment:
🤔 I'm confused. I think there was a reason to document this before.
The content is the same one as bin/jsdoc-to-ts. Would you please figure out just to replace `require` to `import` or so?

Reply:
- I think, `node -e "script"` is the initial method, and since the #12 changes, it has been changed to the CLI method, but it does not appear to be used in the user guide (because it still provides the initial method).
- I searched the way but cannot find... I think import() cannot be used directly in the Bash command line because it is a JavaScript feature used in ESM and is not recognized by Bash. I am concerned that If we should migrate the ESM style, then we need to drop `node -e "script"` usage.